### DISCUSSION TITLE
Help in setting up the project

### DISCUSSION THREAD

**Discussion by Keshav-0907** [ruxailab/RUXAILAB#313] [Q&A] [answered]
- I have followed all the instructions as provided in the README
- I have verified that my .env is correctly configured with firebase credentials 
- I am not getting any errors/warning in console

I successfully signed up as indicated
<img width="1424" alt="Screenshot 2024-02-24 at 11 26 26 AM" src="https://github.com/ruxailab/RUXAILAB/assets/91189139/369bac82-7fb2-41cb-97ec-a2c59b8cbdba">

However, I'm encountering an issue – I'm unable to access the console. I've attached a screenshot for your reference
<img width="509" alt="Screenshot 2024-02-24 at 11 30 59 AM" src="https://github.com/ruxailab/RUXAILAB/assets/91189139/b2a5b567-c5a4-4141-a777-ed5e83baad8f">

I would greatly appreciate your assistance in resolving this matter. 
Please let me know if you need any further information from my end.

**Comment by jvJUCA** [comment, marked as answer]
Dear @Keshav-0907,

Thank you for your report. It appears that the issue may be related to the incorrect configuration of Firebase Functions, potentially affecting the functionality of critical functions like processSignUp.

To ensure smooth operation, we recommend configuring all Firebase Functions properly. This will help guarantee the seamless execution of essential functions throughout your project.

If you require assistance with the configuration process or have any further questions, please don't hesitate to reach out to us.

Thank you for bringing this matter to our attention.

Best regards,

@jvJUCA
